graviti/utility/log.py

```diff
@@ -0,0 +1,268 @@
+# !/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""The implementation of logging utilities."""
+
+import json
+from typing import MutableMapping
+
+from requests.models import PreparedRequest, Response
+from requests_toolbelt.multipart.encoder import FileWrapper, MultipartEncoder
+
+REQUEST_TEMPLATE = """
+===================================================================
+########################## HTTP Request ###########################
+"url": {}
+"method": {}
+"headers": {}
+"body": {}
+"""
+
+RESPONSE_TEMPLATE = """
+########################## HTTP Response ##########################
+"url": {}
+"status_code": {}
+"reason": {}
+"headers": {}
+"content": {}
+"cost_time": {}s
+===================================================================
+"""
+
+
+UPLOAD_DATASET_RESUME_TEMPLATE = """
```

Comment:
The following template is useless in graviti SDK:
- `UPLOAD_DATASET_RESUME_TEMPLATE`
- `UPLOAD_SEGMENT_RESUME_TEMPLATE_SDK`
- `UPLOAD_SEGMENT_RESUME_TEMPLATE_CLI `
- `CACHE_SPACE_WARNING `

Reply:
ok, they have been removed.